addons/website_event/controllers/main.py

```diff
@@ -1,25 +1,19 @@
 # -*- coding: utf-8 -*-
 
-import babel.dates
-import time
-import werkzeug.urls
+import werkzeug
 from datetime import datetime, timedelta
```

Comment:
@WHPODOO  fields.Date class provides only today(strftime), from_string(strptime), to_string(strftime), what if I want to add some days in today
```date.today() + timedelta(days=14)```
Here we did same thing